components/raftstore-v2/src/batch/store.rs

```diff
@@ -490,3 +496,20 @@ where
     };
     (StoreRouter { router, logger }, system)
 }
+
+pub trait MsgRouter: Send {
```

Comment:
It seems only used in tests. I suggest to put it to `local.rs`.

Reply:
Good suggestion.